src/mlpack/methods/ann/layer/lstm.hpp

```diff
@@ -87,6 +87,17 @@ class LSTM
   template<typename InputType, typename OutputType>
   void Forward(InputType&& input, OutputType&& output);
 
+  /**
+   * Ordinary feed forward pass of a neural network, evaluating the function
+   * f(x) by propagating the activity forward through f.
+   *
+   * @param input Input data used for evaluating the specified function.
+   * @param output Resulting output activation.
+   * @param cellState Cell state of LSTM.
```

Comment:
I think we should write of the LSTM here.

Reply:
Done